src/client/apiclient.ts

```diff
@@ -0,0 +1,53 @@
+import {lookupCompanyStatus, lookupCompanyType} from "./api.enumerations";
+import logger from "../logger";
+import { formatDateForDisplay } from "./date.formatter";
+import { createApiClient } from "ch-sdk-node";
+import Resource from "ch-sdk-node/dist/services/resource";
+import { CompanyProfile } from "ch-sdk-node/dist/services/company-profile";
+import { PTFCompanyProfile } from "../model/company.profile";
+
+/**
```

Comment:
Can you run `npm run lint`?
it throws these errors:
ERROR: test/client/api.client.spec.unit.ts:101:1 - Consecutive blank lines are forbidden
ERROR: test/client/api.enumerations.spec.unit.ts:12:18 - Missing trailing comma
ERROR: test/client/api.enumerations.spec.unit.ts:13:14 - Missing semicolon
ERROR: test/client/api.enumerations.spec.unit.ts:14:11 - Missing trailing comma
ERROR: test/client/api.enumerations.spec.unit.ts:15:6 - Missing semicolon